clash-lib/src/Clash/Backend/VHDL.hs

```diff
@@ -1103,6 +1107,7 @@ normaliseType hwty = case hwty of
   -- Simple types, for which a subtype (without qualifiers) will be made in VHDL:
   Clock _           -> Bit
   Reset _           -> Bit
+  Enable _          -> Bit
```

Comment:
`Bool` would be a better choice. `Reset` is `Bit` because it can either be low polarity or high polarity (therefore making True/False potentially very confusing), but there's no such option for enable. As a bonus, that would make this change backwards compatible.

Reply:
You had me at _backwards compatible_